src/components/NewWorkspaceModal.js

```diff
@@ -99,6 +99,9 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
       onSuccess(await Utils.cond(
         [cloneWorkspace, async () => {
           const workspace = await Ajax().Workspaces.workspace(cloneWorkspace.workspace.namespace, cloneWorkspace.workspace.name).clone(body)
+          if (!workspace.googleProject) {
```

Comment:
Please create a variable for the cloudPlatform using `cloneWorkspace.workspace.cloudPlatform` (include the comment at line 109), and use it both here and at line 110.

This will get cleaned up as part of https://broadworkbench.atlassian.net/browse/WOR-723.

Reply:
Or for a more straightforward change, use `isAzureBillingProject` here and leave line 110 alone.

Reply:
Updated!